sys-apps/baselayout/baselayout-2.4.1.ebuild

```diff
@@ -144,6 +144,7 @@ src_prepare() {
 			-e "/PATH=.*\/bin/s|\"$|:/usr/bin:/bin\"|" \
 			etc/profile || die
 		sed -i -r \
+			-e "s@:/bin@:${EPREFIX}/bin@g" \
```

Comment:
@SoapGentoo I could not find a place in the docs that says it's not slash terminated, is it done in portage? In any case, I will push this today, as this is a pretty bad bug to leave as is for too long.

@Polynomial-C I will bump the revision for this, so that users get the update. Let me know if you prefer to handle this differently.